### PR TITLE
Rename toGpuImage to toImageSync

### PR DESCRIPTION
**Pull request by dnfield**
Updates documentation.

Makes toImageSync work in software contexts.

### CODE REVIEW

File: lib/ui/painting/image_encoding.cc [hunk 1]
```diff
@@ -157,11 +157,11 @@ sk_sp<SkData> CopyImageByteData(sk_sp<SkImage> raster_image,
     FML_LOG(ERROR) << "Could not copy pixels from the raster image.";
     return nullptr;
   }
-
+  FML_LOG(ERROR) << "pct: " << pixmap.colorType() << " .. " << color_type;
   // The color types already match. No need to swizzle. Return early.
-  if (pixmap.colorType() == color_type && pixmap.alphaType() == alpha_type) {
-    return SkData::MakeWithCopy(pixmap.addr(), pixmap.computeByteSize());
-  }
+  // if (pixmap.colorType() == color_type && pixmap.alphaType() == alpha_type) {
```
Comment (jonahwilliams) [inline lib/ui/painting/image_encoding.cc:162]: nit: commented code

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: lib/ui/painting/image_encoding.cc (hunk 1 above)
Comment (dnfield) [reply lib/ui/painting/image_encoding.cc]: Whups. Removed the stray log too.

File: shell/common/rasterizer.cc [hunk 2]
```diff
@@ -279,23 +276,22 @@ std::pair<sk_sp<SkImage>, std::string> Rasterizer::MakeGpuImage(
...
-    case MakeGpuImageBehavior::kGpu:
-      break;
-    case MakeGpuImageBehavior::kBitmap:
-      return MakeBitmapImage(picture_size);
-  }
+  const SkImageInfo image_info = SkImageInfo::MakeN32Premul(picture_size);
+  bool gpu_available = true;
+  delegate_.GetIsGpuDisabledSyncSwitch()->Execute(
+      fml::SyncSwitch::Handlers().SetIfTrue(
+          [&gpu_available] { gpu_available = false; }));
```
Comment (ColdPaleLight) [inline shell/common/rasterizer.cc:283]: Can we put the following logic into `SyncSwitch`'s `Handlers`? I guess just using `SyncSwitch` to get the value of `gpu_available` is not enough.

Comment (dnfield) [reply shell/common/rasterizer.cc:283]: Yeah I'll have to refactor this. I realized after uploading

Comment (dnfield) [reply shell/common/rasterizer.cc]: Done

**Pull request merged**